src/main/java/org/terasology/magicalstats/event/BeforeManaRefillEvent.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.magicalstats.event;
+
+import gnu.trove.list.TFloatList;
+import gnu.trove.list.TIntList;
+import gnu.trove.list.array.TFloatArrayList;
+import gnu.trove.list.array.TIntArrayList;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ConsumableEvent;
+/**
+ * Created by monkey on 12/5/16.
```

Comment:
Auto-inserted comment blocks on the other hand we avoid, especially author info since that tends to get copy pasted all over the place and go inaccurate in no time :-)